src/app/organization/offices/edit-office/edit-office.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="officeForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Offices</mat-label>
+            <input matInput required formControlName="name">
+            <mat-error *ngIf="officeForm.controls.name.hasError('required')">
+              Office <strong>is required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Parent Office</mat-label>
```

Comment:
use mat select 
![Screenshot from 2020-05-11 14-04-00](https://user-images.githubusercontent.com/54709463/81541942-85a5d480-9391-11ea-8718-2aa3680acb5c.png)

Reply:
Updated this.